doc/index.rst

```diff
@@ -3,10 +3,26 @@ Julia interface to chemfiles
 
 This is the documentation for the `Julia`_ interface to the `chemfiles`_
 library.
+The `Julia`_ interface to chemfiles wraps around the C interface providing a
+Julian API. All the functionalities are in the ``Chemfiles`` module, which can
+be imported by the ``using Chemfiles`` expression. The ``Chemfiles`` module is
+built around the main types of chemfiles: `Trajectory`_, `Frame`_, `UnitCell`_,
+`Topology`_, `Residue`_, `Atom`_, and `Selection`_.
 
 .. _Julia: http://julialang.org/
 .. _chemfiles: https://github.com/chemfiles/chemfiles
 
+.. warning::
+   All indexing in chemfiles is 0-based! That means that the first atom in a frame
+      have the index 0, not 1. This is because no translation is made from the
+         underlying C library.
+
+         This may change in future release to use 1-based indexing, which is more familiar
+            to Julia developers.
+
+
+
```

Comment:
You need to add the new files in the toctree for them to be picked by sphinx

Reply:
aaaaand, done.